doc/source/whatsnew/v0.23.0.txt

```diff
@@ -697,9 +697,10 @@ Performance Improvements
 - Improved performance of :func:`DataFrame.median` with ``axis=1`` when bottleneck is not installed (:issue:`16468`)
 - Improved performance of :func:`MultiIndex.get_loc` for large indexes, at the cost of a reduction in performance for small ones (:issue:`18519`)
 - Improved performance of pairwise ``.rolling()`` and ``.expanding()`` with ``.cov()`` and ``.corr()`` operations (:issue:`17917`)
-- Improved performance of :func:`DataFrameGroupBy.rank` (:issue:`15779`)
+- Improved performance of :func:`pandas.core.groupby.GroupBy.rank` (:issue:`15779`)
```

Comment:
For whatever reason this one wouldn't render a link. Still debugging but didn't want it to hold up the rest of the review

Reply:
hmm ok

Reply:
it may be that this needs to be DataFrameGroupBy.rank (or maybe SeriesGroupBy.rank)

Reply:
Yea not entirely sure. It was DataFrameGroupBy.rank previously and wasn't rendering, although that was without the entire namespace so perhaps that's it.

Do you want me to open an issue on its own for this or is this typically caught in a "whatsnew cleanup" prior to release?